project_task_subtask_same_project/models/project_task.py

```diff
@@ -24,7 +24,7 @@ def action_subtask(self):
 
         context_vars_to_remove = (
             'search_default_project_id',
-            'search_default_parent_project_ids',  # defined by the module project_iteration
+            'search_default_parent_project_id',  # defined by the module project_iteration
```

Comment:
I would version up the module. What do you think ?

Reply:
Of course

Reply:
done